src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -43,4 +56,56 @@ impl HummockVersionExt for HummockVersion {
             .unwrap_or_else(|| panic!("compaction group {} exists", compaction_group_id))
             .levels
     }
+
+    fn level_delete_ssts(
```

Comment:
If we merge `level_delete_ssts` and `level_insert_ssts` together, we can remove this parameter `l0_remove_position` ?

Reply:
> If we merge `level_delete_ssts` and `level_insert_ssts` together, we can remove this parameter `l0_remove_position` ?

I just created a new function `apply_compact_ssts` without `l0_remove_position` to call them, in order that we do not expose `l0_remove_position` (I also set `level_delete_ssts` and `level_insert_ssts` private)